Assets/Scripts/UI/PauseMenu.cs

```diff
@@ -72,97 +94,139 @@ private void Start()
             currentPanel++;
             numberOfPanels++;
         }
-        for(int x=0;x<4;x++)
+        for (int x = 0; x < 4; x++)
         {
             BuildSettingsUI(x);
         }
         currentPanel = 0;
     }
-    
+
     private void Update()
     {
         if (InputManager.GetButtonDown(PlayerInput.PlayerButton.Pause, player))
         {
-            if (!isPaused && !pauseMenu.activeInHierarchy)
+            if (!isPaused && !main.activeInHierarchy)
             {
                 Pause();
             }
             else if(panels[currentPanel].activeInHierarchy)
             {
                 panels[currentPanel].SetActive(false);
-                pauseMenu.SetActive(true);
+                main.SetActive(true);
                 currentPanel = 0;
 
             }
             else if(settings.activeInHierarchy)
             {
                 settings.SetActive(false);
-                pauseMenu.SetActive(true);
+                main.SetActive(true);
             }
-            else
+            else if(SceneManager.GetActiveScene().name != "Title")
             {
-                ResumeGame();
+                ResumeGame();                
             }
-            EventSystem.current.SetSelectedGameObject(pauseMenu.transform.GetChild(0).gameObject);
+            EventSystem.current.SetSelectedGameObject(main.transform.GetChild(0).gameObject);
         }
     }
+    #region Title Methods
+    public void StartGame()
+    {
+        var latest = GameManager.latestUnlocked;
+        //SceneManager.LoadScene( (latest >= GameManager.totalLevels || latest < 1) ? 1 : GameManager.latestUnlocked);
+        SceneManager.LoadScene(1); // START ON THE FIRST LEVEL FOR DEMO BUILD
```

Comment:
Did I write this? I think I wrote this.
We probably ought to have a *Start* and *Continue* button, but there's not really vertical space for both without resizing.
How difficult!